mindsdb_native/libs/data_types/data_source.py

```diff
@@ -31,10 +31,6 @@ def _setup(self, df, **kwargs):
     def _cleanup(self):
         pass
 
-    @property
-    def df(self):
-        return self._df
-
```

Comment:
mindsb use it in one place: https://github.com/mindsdb/mindsdb/blob/staging/mindsdb/interfaces/datastore/datastore.py#L179

How we should do: restore property in native, or change to ds._df in mindsdb?

Reply:
my bad, I have only checked references to it in native, I will restore the property